sovware/directorist
# fix style compatibility with theme

## PR Type
What kind of change does this PR introduce?


Issues:
- [x]  [https://prnt.sc/yMUQ2a22Q-2R](https://prnt.sc/yMUQ2a22Q-2R)
- [x]  [https://prnt.sc/M3D-umh7-anW](https://prnt.sc/M3D-umh7-anW)
- [x]  [https://prnt.sc/cpP2kSc-jc6B](https://prnt.sc/cpP2kSc-jc6B)
- [x]  [https://prnt.sc/9zhk44v18LJa](https://prnt.sc/9zhk44v18LJa)
- [x]  [https://prnt.sc/c9iq3Huf-q60](https://prnt.sc/c9iq3Huf-q60)
- [x]  [https://prnt.sc/7vTAv2GhhVFH](https://prnt.sc/7vTAv2GhhVFH)
- [x]  [https://prnt.sc/HyDs345y9Hcc](https://prnt.sc/HyDs345y9Hcc)
- [x]  [https://prnt.sc/wy7q1LU5up6m](https://prnt.sc/wy7q1LU5up6m)
- [x]  [https://prnt.sc/aJQNeGJoQrf1](https://prnt.sc/aJQNeGJoQrf1)
- [x]  [https://prnt.sc/lhiyjb47B30g](https://prnt.sc/lhiyjb47B30g)
- [x]  [https://prnt.sc/X4DElX5U1uaa](https://prnt.sc/X4DElX5U1uaa)
- [x]  [https://prnt.sc/fxA-rqEhvCjl](https://prnt.sc/fxA-rqEhvCjl)

How to test & reproduce:
Activate Directorist > Import demo data > Activate 2023 theme > Check all the pages
